src/Microsoft.DotNet.Web.Spa.ProjectTemplates/ReactRedux-CSharp.csproj.in

```diff
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
I guess I'm confused about under what circumstances we need a FrameworkReference? Why do we need it in RazorClassLibrary but not here?

Reply:
The current plan of record is that `Microsoft.NET.Sdk.Web` implies a FrameworkReference to aspnetcore. RazorClassLibrary doesn't use the websdk, it uses Microsoft.NET.Sdk.Razor